pkg/helm/handlers/handler_test.go

```diff
@@ -9,19 +12,100 @@ import (
 	"github.com/openshift/console/pkg/helm/fake"
 )
 
+func FakeHelmHandler() HelmHandlers {
+	return HelmHandlers{
+		GetActionConfigurations: fake.GetActionConfigurations,
+	}
+}
+
 func TestHelmHandlers_HandleHelmList(t *testing.T) {
-	t.Run("List helm releases", func(t *testing.T) {
-		fakeHandlers := HelmHandlers{
-			GetActionConfigurations: fake.GetActionConfigurations,
-			RenderManifests:         fake.RenderManifests,
-			InstallChart:            fake.InstallChart,
-			ListReleases:            fake.ListReleases,
-		}
+	tests := []struct {
+		test        string
+		expectedMsg string
+	}{
+		{test: "invalid test", expectedMsg: "{\"error\":\"Failed to list helm releases: unknown error occurred\"}"},
+		{test: "valid test", expectedMsg: "[{\"name\":\"Test\"}]"},
+	}
+	for _, tt := range tests {
+		t.Run("List helm releases", func(t *testing.T) {
+			handlers := FakeHelmHandler()
+			if tt.test == "valid test" {
+				handlers.ListReleases = fake.ListReleases
+			} else if tt.test == "invalid test" {
+				handlers.ListReleases = fake.ListReleasesFailed
+			}
+
+			request := httptest.NewRequest("", "/foo", strings.NewReader("{}"))
+			response := httptest.NewRecorder()
+
+			handlers.HandleHelmList(&auth.User{}, response, request)
+			switch tt.test {
+			case "valid test":
```

Comment:
Honestly, I would make two different test groups (two test functions) covering valid and invalid cases - it would improve the readability. I would even split tests into different files, say `handler_install_release_test.go`, `handler_list_releases_test.go`, etc.